### PR TITLE
Fix Issue 23873 - [ICE] segfault on imported static if ; else auto x

### PR DESCRIPTION
**Pull request by RazvanN7**

### CODE REVIEW

**Review by thewilsonator — APPROVED** [review]

File: compiler/src/dmd/dimport.d [hunk 1]
```diff
@@ -232,7 +232,21 @@ extern (C++) final class Import : Dsymbol
          * most likely because of parsing errors.
          * Therefore we cannot trust the resulting AST.
          */
-        if (load(sc)) return;
+        import core.stdc.stdio;
```
Comment (dkorpel) [inline compiler/src/dmd/dimport.d:235]: Put this at the top of the module for future debug printfs

Comment (RazvanN7) [reply compiler/src/dmd/dimport.d:235]: sorry, I thought I deleted it.

Comment (maxhaton) [reply compiler/src/dmd/dimport.d]: better still don't include it at all

Comment (dkorpel) [reply compiler/src/dmd/dimport.d]: Walter explicitly asked in an email to keep `import core.stdc.stdio;` imports around for debug `printf`s.

**Pull request merged**